examples/cloudflare-workers/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "solana-action-template",
+  "scripts": {
+    "dev": "wrangler dev",
+    "deploy": "wrangler deploy --minify",
+    "cf-typegen": "wrangler types --env-interface CloudflareBindings"
+  },
+  "dependencies": {
+    "@solana/actions": "^1.0.2",
```

Comment:
Can you update this the `1.0.3`? there was an typo in a constant that has been fixed